src/services/posts/fetch-posts.test.js

```diff
@@ -18,6 +18,7 @@ describe('Fetch all posts', () => {
 	 })
 	 
 	it('should fetch posts', async () =>{
```

Comment:
Please remember that, when new code is added, there have to be test cases which capture the changes made to ensure the quality of the code added.  It helps to know code is working well and also helps to know when the code breaks later. A few questions this pull request should be able to answer in the tests are:

1. is the function returning the correct number of posts when a limit is passed?
2. is the function returning the correct data when a cursor is passed?
3. is the correct response (feed and count) being returned from the function?

Reply:
yes please, I have taken note of this. Thanks